music/src/core/melodies_test.ts

```diff
@@ -58,3 +59,24 @@ test('Test Melody To NoteSequence', (t: test.Test) => {
   t.deepEqual(ns, expected);
   t.end();
 });
+
+test('Test Melody Rhythm Control', (t: test.Test) => {
+  const events = new Int32Array([0, 2, 0, 1, 4, 6, 9, 1]);
+  const melody = new Melody(events, 60, 72);
```

Comment:
i feel this is a bit weird, and you should just be able to pass a normal Javascript array [ ] to that Melody constructor. Are the types getting in the way because it needs to not contain floats?

Reply:
I switched the Melody constructor to take ArrayLike<number>; is that more normal?  I don't want to have it just take number[] because tfjs gives back data as Int32Array.